integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/mediator/test/smooks/SmooksLargeFileProcessingTestCase.java

```diff
@@ -0,0 +1,127 @@
+/*
+*Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
The licence needs to be updated

Reply:
Actually I renamed an existing class therefore I'm not sure if its correct to change the license year.